packages/sanity/src/core/form/inputs/ReferenceInput/useReferenceInput.tsx

```diff
@@ -2,7 +2,7 @@ import React, {ComponentProps, ForwardedRef, forwardRef, useCallback, useMemo, u
 import {from, throwError} from 'rxjs'
 import {catchError, mergeMap} from 'rxjs/operators'
 import * as PathUtils from '@sanity/util/paths'
-import {
+import type {
```

Comment:
on an completely unrelated note, we might want to add an eslint rule to enforce this consistently throughout the codebase